### PR TITLE
Add CloudCasa to "Adopters of Velero" section on velero.io

### PR DESCRIPTION
**Pull request by kkothule**
Thank you for contributing to Velero!

# Please add a summary of your change

- Added CloudCasa logo
- Added description of CloudCasa and its integration with Velero

Signed-off-by: Kiran Kothule from Catalogic Software <kkothule@catalogicsoftware.com>

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: ADOPTERS.md
Comment (kaovilai) [inline ADOPTERS.md:61]: image

Might want to check the resulting formatting. Shouldn't there be a new line before org name?

**Review by kaovilai — CHANGES_REQUESTED** [review]

**Comment by kkothule** [comment]
@kaovilai please review the changes

**Review by kaovilai — APPROVED** [review]
Not a maintainer but lgtm

**Comment by kkothule** [comment]
Thanks, @kaovilai, Please help me in getting the final review from the maintainer, Whom should I assign PR for review or if you can assign it to someone

**Comment by kaovilai** [comment]
Technically it's already assigned. Just wait a bit.

**Comment by a-mccarthy** [comment]
Thanks for submitting this @kkothule! This lgtm, tagging @jonasrosland and @OrlinVasilev for review as well

**Comment by jonasrosland** [comment]
This LGTM, thank you @kkothule !

**Review by sseago — APPROVED** [review]

**Comment by kkothule** [comment]
Thank You @jonasrosland and @a-mccarthy, Can we merge this to the main branch?

**Review by OrlinVasilev — APPROVED** [review]
LGTM :)

**Comment by kkothule** [comment]
Thanks @jenting

**Pull request merged**
